### DISCUSSION TITLE
How about a TDP option, or option to limit battery discharge

### DISCUSSION THREAD

**Discussion by jhuddle1** [seerge/g-helper#1799] [Ideas]
is there any way if its not added to limit tdp on intel processors like on amd processors ? and any way to limit the discharge rate to save battery ? also undervolting cpu option

**Comment by seerge** [comment]
There is an option to limit power on Intel CPUs as well :) If you had that option in Armoury - you would see it on G-Helper as well (under Fans+Power), if you  didn't - your device doesn't support that.

As for undervolting - check BIOS (F2 on boot) or Inte's own utility https://www.intel.com/content/www/us/en/download/17881/intel-extreme-tuning-utility-intel-xtu.html
